ci/includes/pinned-tools.sh

```diff
@@ -31,3 +38,13 @@ export LINTER="cleanupcode.exe"
 export MSBUILD="$(powershell  –ExecutionPolicy Bypass ./ci/find-msbuild.ps1)"
 
 export NUNIT3_CONSOLE="code_generator/packages/NUnit.ConsoleRunner.3.8.0/tools/nunit3-console.exe"
+
+function unpackTo() {
+  local SOURCE=$1
+  local TARGET=$2
+
+  mkdir -p "${TARGET}"
+  unzip -o -q "${SOURCE}" -d "${TARGET}"
```

Comment:
Including a dependency on unzip - may need to include in setup guide

Reply:
It's included in git bash, so since we already depend on that, it should be fine for a bit longer?